build/RELEASING.md

```diff
@@ -0,0 +1,97 @@
+
+# Release Process
+
+Releasing of projects from the opentelemetry-dotnet-contrib repo is
+currently a mix of automated workflow and some manual steps. Hopefully
+most of this will be automated soon.
+
+## Pre-requisites
+
+To carry out the release successfully, make sure you fulfill the following:
+
+1. You must be a maintainer on the repo. This will allow you to push tags to
+`main` branch and create a GitHub release later on.
+
+2. Have access to the Nuget token for releasing the project to Nuget.org
+OpenTelemetry account.
+
+## Pre-steps
+
+1. Update the Changelog for your project with relevant details. You
```

Comment:
How about:
1. Decide on the version to be used.
2. Update changelog to replace Unreleased with this version and add date of release.
3. Submit PR to update changelog and get it merged.

and in the Steps.1 - mention that the version should be same as the one updated in changelog.